working-groups/parallel-rustc/README.md

```diff
@@ -0,0 +1,43 @@
+# parallel-rustc Working Group
```

Comment:
Can you also add this to the index table of the repo?

I'm going to add that to the instructions in the template as it has happened a few times and is a failing of the template.